app/models/comment.rb

```diff
@@ -3,4 +3,8 @@ class Comment < ApplicationRecord
   belongs_to :post
 
   validates :text, presence: true
+
```

Comment:
It would be nice to add the after_save :update_comments_counter method to update the database after a comment has been added 👍🏿